app/controllers/api/v1/visits_controller.rb

```diff
@@ -1,14 +1,39 @@
 module Api
   module V1
     class VisitsController < ApplicationController
-
       before_action :set_visit, only: %i[show update destroy]
 
       attr_accessor :visit
 
       def index
-        @visits = Visit.all
-        render json: @visits
+        if params[:sort]
+          sort_params = params[:sort]
+
+          if sort_params[:start_at]
+            sort_field = :start_at
+            sort_order = sort_params[:start_at]
+          end
+
+          if sort_params[:end_at]
+            sort_field = :end_at
+            sort_order = sort_params[:end_at]
+          end
+
+          if sort_params[:adress]
+            sort_field = :adress
+            sort_order = sort_params[:adress]
+          end
+
+          if sort_params[:price]
+            sort_field = :price
+            sort_order = sort_params[:price]
+          end
+          @visits = Visit.order(sort_field => sort_order)
+          render json: @visits
+        else
+          @visits = Visit.all
+          render json: @visits
+        end
```

Comment:
as far as i understand, this will do the same sorting:
render json: params[:sort] ? Visit.order(params[:sort]) : Visit.all